bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -173,4 +173,11 @@
 		<state readOnly="true"/>
 		<config-description-ref uri="channel-type:tacmi:schemaApiDefaults"/>
 	</channel-type>
+	<channel-type id="schema-date-time-ro">
+		<item-type>DateTime</item-type>
+		<label>Value</label>
```

Comment:
Not sure in exact what cases it is used, but i guess it can be a little bit more specific
```suggestion
		<label>DateTime Value</label>
```